apis/infix/atrium-api-infix/src/commonTest/kotlin/ch/tutteli/atrium/api/infix/en_GB/CollectionExpectationsTest.kt

```diff
@@ -15,12 +17,13 @@ class CollectionExpectationsSpec : ch.tutteli.atrium.specs.integration.Collectio
 
     }
 
-    @Suppress("unused", "UNUSED_VALUE")
-    private fun ambiguityTest() {
-        var a1: Expect<List<Int>> = notImplemented()
-        var a1b: Expect<Set<Int?>> = notImplemented()
+    @Suppress("AssignedValueIsNeverRead")
+    @Test
+    fun ambiguityTest() {
+        var a1: Expect<List<Int>> = expect(listOf(1))
+        var a1b: Expect<Set<Int?>> = expect(setOf(1))
 
-        var star: Expect<Collection<*>> = notImplemented()
+        var star: Expect<Collection<*>> = expect(listOf(1))
 
         a1 toBe empty
         a1 notToBe empty
```

Comment:
this one will surely fail, a1 cannot be empty and non-empty at the same time :wink:

Reply:
@robstoll - I just fixed all of those tests and pushed a commit before the rebasing. I tested them locally and they pass.